src/Symfony/Component/ErrorHandler/Error/FatalError.php

```diff
@@ -33,7 +33,8 @@ public function __construct(string $message, int $code, array $error, ?int $trac
                 }
             }
         } elseif (null !== $traceOffset) {
-            if (\function_exists('xdebug_get_function_stack') && $trace = @xdebug_get_function_stack()) {
+            if (\function_exists('xdebug_get_function_stack') && \in_array(\ini_get('xdebug.mode'), ['develop', false], true)) {
+                $trace = xdebug_get_function_stack();
```

Comment:
let's do it this way to be extra sure, esp with older versions?
```suggestion
            if (\function_exists('xdebug_get_function_stack') && \in_array(\ini_get('xdebug.mode'), ['develop', false], true) && $trace = @xdebug_get_function_stack();
```

Reply:
Done